rust/src/bin/mina-indexer.rs

```diff
@@ -171,13 +185,18 @@ pub async fn main() -> anyhow::Result<()> {
     match cli.command {
         IndexerCommand::Client(args) => client::run(&args, &domain_socket_path).await,
         IndexerCommand::Server { server_command } => {
-            let (args, mut mode) = match server_command {
+            let (args, mut mode) = match *server_command {
                 ServerCommand::Shutdown => {
                     return client::run(&client::ClientCli::Shutdown, &domain_socket_path).await;
                 }
                 ServerCommand::Start(args) => (args, InitializationMode::New),
                 ServerCommand::Sync(args) => (args, InitializationMode::Sync),
                 ServerCommand::Replay(args) => (args, InitializationMode::Replay),
+                ServerCommand::Config(args) => {
```

Comment:
It seems that this is at the wrong level of hierarchy for commands? 'Config' seems to also imply 'start'? Is that intended? If so, maybe this should be called "start-with-config"?

Reply:
`start` and `config` are essentially the same; `start` gets CLI args explicitly, `config` gets them from a file